packages/sdk/src/chains/Substrate/assetTransfer.ts

```diff
@@ -48,8 +48,8 @@ export class SubstrateAssetTransfer {
 
   public async init(
     apiPromise: ApiPromise,
-    environment: Environment = Environment.MAINNET,
-    parachainId: SubstrateParachain = SubstrateParachain.LOCAL,
+    environment: Environment,
```

Comment:
I would leave this as default and move it below parachainId

Reply:
Done